app/assets/sass/application.scss

```diff
@@ -48,6 +48,11 @@ tr.shown td.details-control {
   background: url('../images/details_close.png') no-repeat center center;
 }
 
+.capacity-red {
```

Comment:
Moved this style out of the table styles as it is now used in the text above the table